src/views/Events/index.jsx

```diff
@@ -0,0 +1,207 @@
+    end: new Date(new Date().setHours(new Date().getHours() + 3)),
+    description: 'Near to today',
+    location: 'here',
+    imageUrl:
+      'https://images.unsplash.com/photo-1524758631624-e2822e304c36?ixlib=rb-1.2.1&ixid=MnwxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
+  },
+  {
+    id: 4,
+    title: 'Tomorrow',
+    start: new Date(new Date().setDate(new Date().getDate() + 1)),
+    end: new Date(new Date().setDate(new Date().getDate() + 1)),
+    description: 'Near to tomorrow',
+    location: 'nus',
+  },
+  {
+    id: 12,
+    title: 'XIt will be there yeah',
+    location: 'NUS School of Computing',
+    date: new Date('01/27/2022'),
+    description:
+      ':S DjJ ;lakJD;flkds;lfja;mx clkvnsghsdfja;dlskjfa;dlkj fa;lkjd ;alknvlcnva;lsdn fasdf;laskjd f;alkmvlxcnv;',
+    allDay: true,
+    start: new Date('01/27/2022 01:00'),
+    end: new Date('01/27/2022 02:00'),
+    imageUrl:
+      'https://images.unsplash.com/photo-1643037906067-e79f62e8f58c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=687&q=80',
+  },
+  {
+    id: 13,
+    title: 'XToday',
+    start: new Date(new Date().setHours(new Date().getHours() - 3)),
+    end: new Date(new Date().setHours(new Date().getHours() + 3)),
+    description: 'Near to today',
+    location: 'here',
+    imageUrl:
+      'https://images.unsplash.com/photo-1524758631624-e2822e304c36?ixlib=rb-1.2.1&ixid=MnwxMjA3fDF8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80',
+  },
+  {
+    id: 14,
+    title: 'XTomorrow',
+    start: new Date(new Date().setDate(new Date().getDate() + 1)),
+    end: new Date(new Date().setDate(new Date().getDate() + 1)),
+    description: 'Near to tomorrow',
+    location: 'nus',
+  },
+]
+
+const Events = () => {
+  const classes = useStyles()
+
+  console.log(events.at(1))
+
+  return (
+    <Layout>
+      <Container maxWidth="lg">
+        <Typography
+          variant="h4"
+          style={{
+            color: '#4B87B1',
+            textAlign: 'center',
+            display: 'block',
+            marginTop: '80px',
+            marginBottom: '40px',
+          }}
+        >
+          Upcoming Events
+        </Typography>
+        <Box className={classes.spotlight}>
+          <Box className="calendar">
+            {events && (
+              <Calendar
+                views={['day', 'month', 'week']}
+                events={events.map((a, i) => ({
+                  id: i,
+                  title: a.title,
+                  start: a.start,
+                  end: a.end,
+                }))}
+                step={60}
+                defaultDate={new Date()}
+                localizer={localizer}
+                popup={true}
+                startAccessor="start"
+                endAccessor="end"
+                titleAccessor="title"
+              />
+            )}
+          </Box>
+          <Box className="event">
```

Comment:
Oh question, do you intend to include logic to dynamically display the spotlight and various events? Seems like now the spotlight is the first in our event records and and the various events follow.

I was thinking maybe can consider:
- For each day / month / week view, we only show events that are during that time frame. The spotlight will default be the earliest upcoming event
- Alternatively, we can show only events that have not been completed. The spotlight will default be the earliest upcoming event

Reply:
I think the second one will be better. The spotlight is the earliest event.